doc/source/dev/contributor/meson.rst

```diff
@@ -137,6 +137,42 @@ To run the tests for a particular submodule(let's say ``optimize``), you can use
   python dev.py -n -s optimize
 
 
+Use GCC and Clang build parallely
+=================================
+
+As discussed, Meson is fully out-of-place, so different builds will not interfere
+with each other. For example, let us build using GCC and Clang.
+
+1. Build with GCC::
+
+    export CC=gcc
+    meson setup build-gcc --prefix=$PWD/install-gcc
+    ninja -C build-gcc
```

Comment:
Why not use `python dev.py` here? We no longer need the separate `setup`/`build`/`install` steps.